### ISSUE TITLE
"Convert MS Word to AsciiDoc" content discussion

### ISSUE THREAD

**Issue by rockyallen** [asciidoctor/asciidoctor.org#542]
Use this issue as a tracker for content suggestions, or feel free to provide PRs referencing this ticket to keep track of contributions.

This content is contained in https://github.com/asciidoctor/asciidoctor.org/blob/master/docs/user-manual.adoc.

**Comment by rockyallen** [comment]
Source material:
* http://discuss.asciidoctor.org/My-experience-using-Asciidoctor-with-a-windows-documentation-toolchain-tt4443.html#a4486
* http://thepaulrayner.com/blog/2013/02/14/word-to-asciidoc/
* Pandoc .docx to .adoc
* Word->HTML->AsciiDocFx

**Comment by mojavelinux** [comment]
There's also a plugin for Google Docs by @Mogztter for converting a rich text document to AsciiDoc. It's still in the early stages, but could prove useful to some.

* https://github.com/Mogztter/asciidoc-googledocs-addon

**Comment by rockyallen** [comment]
So far I agree with http://discuss.asciidoctor.org/My-experience-using-Asciidoctor-with-a-windows-documentation-toolchain-tt4443.html#a4486. 
docx->pandoc get most of the way there. I have a few questions on the pandoc discussion to confirm the best process. html->pandoc is no better, and you have to fix the encoding of the html first. If anyone wants to review AsciidoctorFx and Google Docs, do we have enough to draft the section?
Also, is it a good idea to start a branch in the Asciidoctor repository (if that is possible) to make it easier to collaborate on?

**Comment by mojavelinux** [comment]
I don't think AsciidocFX is pertinent for this section (though certainly deserves a mention in the convert HTML to AsciiDoc section).

We can probably take content from the README of the AsciiDoc add-on for Google Docs. We should make it clear that it's not yet a comprehensive solution, but could prove useful for simpler docs.

>  is it a good idea to start a branch in the Asciidoctor repository

While GitHub is awesome for forks, it's not very good for collaborative branches. It requires that everyone participating have write access to the main repository, which can lead to problems.

The best way to work is for at least one person (the PR owner) to send a pull request to the main repository. Other collaborators can then send a pull request to that branch with additional changes. The PR owner has to merge those additional changes into that branch.

In other words, a PR is a way to start a branch. The branch just happens to sit in the fork instead of the upstream.

I suppose we could reserve a branch in the main repository to receive these pull requests. That would make it a bit simpler, but it's basically the same idea.

**Comment by rockyallen** [comment]
See
https://github.com/rockyallen/asciidoctoressays/blob/master/wordimport/build.xml
(finally)

**Comment by mojavelinux** [comment]
@rockyallen Would you like to add this to the new guide?

http://asciidoctor.org/docs/migrating-from-msword/

**Comment by mojavelinux** [comment]
Could you submit a PR?
